projects/stakingverse/index.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Stakingverse adapter for DefiLlama
+ * Tracks TVL for LUKSO staking 
+ */
+
+const STAKING_CONTRACT = '0x9F49a95b0c3c9e2A6c77a16C177928294c0F6F04';
+const SLYX_TOKEN_CONTRACT = '0x8a3982f0a7d154d11a5f43eec7f50e52ebbc8f7d';
+const LYX_ADDRESS = '0x0000000000000000000000000000000000000000'; // Native token null address
+
+/**
+ * TVL function - calculates the total value of LYX staked in the staking contract
+ */
+async function tvl(api) {
+  // Get total staked LYX using totalAssets() method
+  const totalStakedLYX = await api.call({
+    target: STAKING_CONTRACT,
+    abi: 'function totalAssets() view returns (uint256)'
+  });
+
+  // Add native LYX to TVL
+  api.add(LYX_ADDRESS, totalStakedLYX);
+
+  return api.getBalances();
+}
+
+/**
+ * Pool2 function - calculates the total value of liquid staking tokens (sLYX)
+ * converted to their underlying LYX value based on the current exchange rate
+ */
+async function pool2(api) {
```

Comment:
please remove the pool2 function, this is double counting the underlying LYX

Reply:
I did that to account the TVL of the LST and pool separately, I will change it if you are sure this is needed to be deleted.

![Screenshot 2025-04-23 at 2 55 53 PM](https://github.com/user-attachments/assets/8081c220-921f-4ac9-8a2e-27d03cc9a943)